index.js

```diff
@@ -45,7 +45,14 @@ const server = http.createServer(async function (request, response) {
                 grid_type: 'grid',
                 size: params.renderMethod === 'hits' ? 10000 : 0,// only populate the hits layer when necessary
                 track_total_hits: false,
-                body: {}
+                body: params.searchQuery ? {
```

Comment:
Now that you are setting body, would it makes sense to move all parameters there?

Reply:
Yep, seems better to put all the keys inside `body` following [the documentation](https://www.elastic.co/guide/en/elasticsearch/reference/master/search-vector-tile-api.html#search-vector-tile-api-request-body) and conditionally add just the `query` property.

I'll improve this tomorrow, thanks!

Reply:
In [7fb3ee0](https://github.com/thomasneirynck/mvt_sample/pull/5/commits/7fb3ee0df0326ed0767a20024c74bf744eaf9673) I've moved the query parameters to the `body` and added a explicit query to retrieve everything if there's nothing to search.

Reply:
Yep, done, sorry I missed those two from the docs.